buf.gen.datasets.yaml

```diff
@@ -14,3 +14,4 @@ inputs:
   - module: buf.build/tilebox/api
     paths:
       - "datasets"
+      - "tilebox"
```

Comment:
I guess there is no way to have all three modules in one file and write them into different packages?

If that's the case let's leave a comment above at the plugins to make sure to always keep `datasets.yaml` in sync with `workflows.yaml` when it comes to plugin version numbers.